csharp/src/Microsoft.Azure.EventHubs/Primitives/EventHubsConnectionSettings.cs

```diff
@@ -92,7 +92,7 @@ public EventHubsConnectionSettings(string namespaceName, string entityPath, stri
         /// Endpoint=sb://namespace_DNS_Name;EntityPath=EVENT_HUB_NAME;SharedAccessKeyName=SHARED_ACCESS_KEY_NAME;SharedAccessKey=SHARED_ACCESS_KEY
         /// </summary>
         /// <param name="connectionString">Event Hubs ConnectionString</param>
-        public EventHubsConnectionSettings(string connectionString)
+        public EventHubsConnectionSettings(string connectionString, string entityPath = null)
```

Comment:
>string entityPath = null [](start = 68, length = 24)

I believe we should also throw if there is already an entity path in the connection string.